docs/tutorials/getting-started-with-csharp/microservices.md

```diff
@@ -106,32 +108,21 @@ browser and navigate to that page and see a "Hello World!" message.
 
 Now that you've built the application, let's look at how this functionality
 is implemented. There are two of the generated files that are particularly
-interesting at this point: project.json and startup.cs. 
+interesting at this point: project.json and Startup.cs. 
 
-Project.json contains information about the project. The three nodes you'll
-often work with are 'dependencies', 'commands' and 'frameworks'. The
+Project.json contains information about the project. The two nodes you'll
+often work with are 'dependencies' and 'frameworks'. The
 dependencies node lists all the packages that are needed for this application.
 At the moment, this is a small node, needing only the packages that run the
 web server.
 
-The 'commands' node lists the command that runs the web server. As a project
-grows, you'll add commands here to run unit tests, or perform other tasks. The
-web task starts the webserver.
-
 The 'frameworks' node specifies the versions and configurations of the .NET
 framework that will run this application.
 
 The application is implemented in Startup.cs. This file contains the startup
-class. Its `Main()` method starts the web server, and instructs the web
-server that the web application class is this Startup class:
-
-`public static void Main(string[] args) => Microsoft.AspNet.Hosting.WebApplication.Run<Startup>(args);`
-
-This method is a simple one-line method, so it utilizes the C# syntax for
```

Comment:
I removed this, because `Main` looks differently in RC2:

```C#
public static void Main()
{
    var host = new WebHostBuilder()
        .UseKestrel()
        .UseContentRoot(Directory.GetCurrentDirectory())
        .UseIISIntegration()
        .UseStartup<Startup>()
        .Build();

    host.Run();
}
```

I thought that explaining all of that would be outside the scope of this article.

(And it could be an expression-bodied member, but I'm not sure that's a good idea here, since it's not just one or two lines.)